docs/rest_api.rst

```diff
@@ -71,862 +74,584 @@ Following are the available API endpoints with which you can interact with Raide
 Querying Information About Your Raiden Node
 ===============================================
 
-Querying your address
----------------------
+.. http:get:: /api/(version)/address
 
-When raiden starts you choose an ethereum address which will also be your raiden address. You can query that address by making a ``GET`` request to the ``/api/<version>/address`` endpoint.
+   Query your address. When raiden starts, you choose an ethereum address which will also be your raiden address.
 
-Example Request
-^^^^^^^^^^^^^^^
+   **Example Request**:
```

Comment:
Why change from subheading to bold here and below?

Reply:
Subheadings do not work within the `sphinxcontrib.httpdomain` directives, and the bold headings are consistent with the generated headings. (see example rendering below)

Reply:
Okay that looks way cooler. Hands down.